_docs/principles/accessibility.md

```diff
@@ -264,18 +379,19 @@ In this example, we provide a clear visual feedback that the focus has moved to
 
 <div class="hxRow"  markdown="1">
 
-{% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-12-md hxSpan-12-lg" %}
+{% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-8-md hxSpan-8-lg" %}
 
-### Tools
+## Resources
 
-These best practices are not exhaustive, and they are not necessarily applicable to every situation. However, they do cover the major areas for concern when designing for visual accessibility. It is more important to understand the principles so that you can integrate them into your design processes. Ultimately, improving accessibility benefits all users by providing greater ease of use to everyone.
+These accessibility guidelines are not exhaustive, and they might not apply to every situation. However, they do cover the major areas of concern when you incorporate accessibility into your design. Improving accessibility benefits all users by providing greater ease of use to everyone.
 
-There are many tools that can help you with accessibility. Check them out to learn more:
+Consult the following resources as necessary:
 
-- [WebAim’s color contrast checker](https://webaim.org/resources/contrastchecker/): Insert foreground and background colors to know if they pass accessibility guidelines.
-- [Toggle Grayscale](https://goo.gl/aFn2h4): Plug-in built by the Helix development team to quickly view prototypes published to the web in grayscale within the Chrome browser.
-- [I want to see like the color blind](https://chrome.google.com/webstore/detail/i-want-to-see-like-the-co/jebeedfnielkcjlcokhiobodkjjpbjia?hl=en-GB): Plug-in built to apply filters specific to color deficiencies. Usable only for Chrome.
-- [How to convert a whole design to grayscale](http://www.sketchtips.info/articles/how-to-convert-a-whole-design-to-grayscale): Guide to quickly apply grayscale to designs from within Sketch.
+- [How to Meet WCAG 2 (Quick Reference)](https://www.w3.org/WAI/WCAG21/quickref/): A quick reference guide that contains accessibility requirements success criteria and techniques.
+- [WebAim's color contrast checker](https://webaim.org/resources/contrastchecker/): Insert foreground and background colors to know if they pass accessibility guidelines.
+- [Toggle Grayscale](https://goo.gl/aFn2h4): A plug-in built by the Helix development team so you can quickly view prototypes published to the web in grayscale within the Chrome browser.
```

Comment:
`A Google Chrome extension built by the Helix development team that allows you to quickly view web content in grayscale.`

Reply:
Done.